mLRS/Common/setup.h

```diff
@@ -28,15 +28,19 @@ void setup_configure_metadata(void)
 {
     SetupMetaData = {0};
 
-    //-- FrequencyBand: "2.4,915 FCC,868"
+    //-- FrequencyBand: "2.4,915 FCC,868,433,70 cm"
 #ifdef FREQUENCY_BAND_2P4_GHZ
-    SetupMetaData.FrequencyBand_allowed_mask = 0b0001; // only 2.4 GHz, not editable
+    SetupMetaData.FrequencyBand_allowed_mask = 0b00001; // only 2.4 GHz, not editable
 #elif defined FREQUENCY_BAND_915_MHZ_FCC && defined FREQUENCY_BAND_868_MHZ
-    SetupMetaData.FrequencyBand_allowed_mask = 0b0110; // only 915 FCC, 868
+    SetupMetaData.FrequencyBand_allowed_mask = 0b00110; // only 915 FCC, 868
 #elif defined FREQUENCY_BAND_915_MHZ_FCC
-    SetupMetaData.FrequencyBand_allowed_mask = 0b0010; // only 915 MHz FCC, not editable
+    SetupMetaData.FrequencyBand_allowed_mask = 0b00010; // only 915 MHz FCC, not editable
 #elif defined FREQUENCY_BAND_868_MHZ
-    SetupMetaData.FrequencyBand_allowed_mask = 0b0100; // only 868 MHz, not editable
+    SetupMetaData.FrequencyBand_allowed_mask = 0b00100; // only 868 MHz, not editable
+#elif defined FREQUENCY_BAND_433_MHZ
```

Comment:
do we want the option to have both 433 and 70?
could it not even make sense to allow the option of having all four, 868,915,433,70?
e.g., on diy modules one may want to have that, so that one doesn't need a new firmware for a different E22
keep it as is, I'm just curious about your thoughts

Reply:
Not a bad idea to have both 433 & 70.  Have added this.